diff --git a/node_modules/@mapbox/tiny-sdf/index.js b/node_modules/@mapbox/tiny-sdf/index.js
index 5a56c46..af0e003 100644
--- a/node_modules/@mapbox/tiny-sdf/index.js
+++ b/node_modules/@mapbox/tiny-sdf/index.js
@@ -55,7 +55,15 @@ export default class TinySDF {
         const glyphLeft = 0;
 
         // If the glyph overflows the canvas size, it will be clipped at the bottom/right
-        const glyphWidth = Math.max(0, Math.min(this.size - this.buffer, Math.ceil(actualBoundingBoxRight - actualBoundingBoxLeft)));
+        // ADDING EXTRA PADDING otherwise "j" will be cut off
+        const EXTRA_PADDING = 2;
+        const glyphWidth = Math.max(
+        0,
+        Math.min(
+            this.size - this.buffer,
+            Math.ceil(actualBoundingBoxRight - actualBoundingBoxLeft) + EXTRA_PADDING,
+        ),
+        );
         const glyphHeight = Math.min(this.size - this.buffer, glyphTop + Math.ceil(actualBoundingBoxDescent));
 
         const width = glyphWidth + 2 * this.buffer;
